src/content/configuration/dev-server.md

```diff
@@ -1226,3 +1226,36 @@ module.exports = {
 If this is too heavy on the file system, you can change this to an integer to set the interval in milliseconds.
 
 See [WatchOptions](/configuration/watch/) for more options.
+
+
+## `devServer.writeToDisk` 🔑
+
+`boolean` `function`
```

Comment:
please add default value here and remove below. please add function signature here e.g. `function (param1, param2)`

Reply:
Done. thank you